mono/mini/Makefile.am.in

```diff
@@ -836,8 +847,8 @@ llvmonlycheck: mono $(llvmonly_regtests)
 	rm -rf fullaot-tmp
 	mkdir fullaot-tmp
 	$(MAKE) fullaot-libs AOT_FLAGS="llvmonly,$(MONO_FULLAOT_ADDITIONAL_ARGS)$(INVARIANT_AOT_OPTIONS)"
-	cp $(llvmonly_regtests) generics-variant-types.dll TestDriver.dll fullaot-tmp/
-	MONO_PATH=fullaot-tmp $(top_builddir)/runtime/mono-wrapper  $(MOBILE_RUNTIME_ARG) --aot=llvmonly fullaot-tmp/{generics-variant-types.dll,TestDriver.dll,*.exe} || exit 1
+	cp $(llvmonly_regtests) $(ADDITIONAL_MINI_TEST_DEPS) fullaot-tmp/
+	MONO_PATH=fullaot-tmp $(top_builddir)/runtime/mono-wrapper  $(MOBILE_RUNTIME_ARG) --aot=llvmonly fullaot-tmp/{$(ADDITIONAL_MINI_TEST_DEPS_COMMAS),*.exe} || exit 1
```

Comment:
wouldn't it be easier to use `$(addprefix fullaot-tmp/,$(ADDITIONAL_MINI_TEST_DEPS) *.exe)` instead of the comma makefile trickery to get the bash syntax?